Core/src/org/sleuthkit/autopsy/discovery/search/DiscoveryAttributes.java

```diff
@@ -280,6 +278,21 @@ private void processResultFilesForCR(List<Result> results,
                             currentFiles.clear();
                     }
                 } else {
+                    ResultDomain domain = (ResultDomain) result;
```

Comment:
This seems like a mistake with a merge. This was the first method I wrote, but then I improved it later. The state of the domain-discovery branch is correct https://github.com/sleuthkit/autopsy/blob/domain-discovery/Core/src/org/sleuthkit/autopsy/discovery/search/DiscoveryAttributes.java#L255. 

You can just remove all this code.

Reply:
good catch will fix.